c2corg_ui/static/js/maincontroller.js

```diff
@@ -78,9 +78,9 @@ app.MainController.prototype.isPath = function(path) {
     return 'home';
   // if topoguide, it can be all kinds of documents
   } else if (path === 'topoguide') {
-    return location.indexOf('waypoints') > -1 || location.indexOf('routes') > -1 || location.indexOf('outings') > -1
-            || location.indexOf('areas') > -1 || location.indexOf('images') > -1 || location.indexOf('books')
```

Comment:
Changing the tests order is not relevant to me when the only required change was to add the missing ``> -1``.
I admit it is probably peanuts but it's better to test the most usual document first (waypoints, routes, outings). Book pages are more rarely used.